extensions/podman/package.json

```diff
@@ -120,6 +120,12 @@
           "scope": "ContainerProviderConnectionFactory",
           "description": "Start the machine now",
           "when": "podman.isStartNowAtMachineInitSupported"
+        },
+        "podman.setting.dockerCompatibility": {
+          "type": "boolean",
+          "default": false,
+          "description": "Docker compatibility mode is a feature that allows you to use Podman as a drop-in replacement for Docker-compatible CLI tools. Podman will automatically emulate a Docker API socket connection (typically) to /var/run/docker.sock. NOTE: This requires administrative privileges.",
+          "when": "isMac || isLinux"
```

Comment:
This is not supported i guess. I can see the setting on Windows 😆

Reply:
Okay I'll open up an issue and work on it now.